### PR TITLE
Fix Eclipse type-safety warnings in overrides of `ModRef` visitor-making methods

### PR DESCRIPTION
**Pull request by liblit**

### CODE REVIEW

File: com.ibm.wala.core/src/com/ibm/wala/ipa/modref/ModRef.java [hunk 1]
```diff
@@ -342,7 +342,7 @@ protected ModVisitor makeModVisitor(CGNode n, Collection<PointerKey> result, Poi
     return makeModVisitor(n, result, pa, h, false);
   }
 
-  protected ModVisitor<T,? extends ExtendedHeapModel> makeModVisitor(CGNode n, Collection<PointerKey> result, PointerAnalysis<T> pa, ExtendedHeapModel h,
+  protected ModVisitor<T, ? extends ExtendedHeapModel> makeModVisitor(CGNode n, Collection<PointerKey> result, PointerAnalysis<T> pa, ExtendedHeapModel h,
```
Comment (msridhar) [inline com.ibm.wala.core/src/com/ibm/wala/ipa/modref/ModRef.java]: This was [just changed](https://github.com/wala/WALA/pull/172/files#diff-409e334c66c9e19b63fba57e6cb66973R345) by @khatchad to remove the `? extends`.  @khatchad does the wildcard cause a problem?

Comment (liblit) [inline com.ibm.wala.core/src/com/ibm/wala/ipa/modref/ModRef.java]: Are you sure @khatchad removed that wildcard? The [link you provided](https://github.com/wala/WALA/pull/172/files#diff-409e334c66c9e19b63fba57e6cb66973R345) looks to me like it's *adding* wildcards in two places (lines 345 and 371), but not removing any. Or am I looking at the diff backwards?

Comment (msridhar) [inline com.ibm.wala.core/src/com/ibm/wala/ipa/modref/ModRef.java]: Yeah you're right I think. Can you rebase then to make the changes clearer?

Comment (liblit) [inline com.ibm.wala.core/src/com/ibm/wala/ipa/modref/ModRef.java]: Now rebased, I *think*. (I don't rebase much.)

**Pull request merged**